browser/src/control/Control.Menubar.js

```diff
@@ -1636,6 +1640,8 @@ L.Control.Menubar = L.Control.extend({
 								$(aItem).removeClass(constChecked);
 							}
 						}
+					} else if ((id.startsWith('zotero') && !(self._map.zotero && self._map.zotero.enable))) {
+						$(aItem).addClass('disabled');
```

Comment:
@timar there was conflicts due to recent references related PR merges. I have rebased. But now the menu entries do not get disabled if zotero is not enabled maybe something from https://github.com/CollaboraOnline/online/pull/5582 impacted this?

Reply:
Probably, it's cool that @eszkadev fixed and merged my PR. I'll take care of this PR next, don't worry.